hbase-client/src/main/java/org/apache/hadoop/hbase/client/MetricsConnection.java

```diff
@@ -457,6 +525,27 @@ public void incrementServerOverloadedBackoffTime(long time, TimeUnit timeUnit) {
     overloadedBackoffTimer.update(time, timeUnit);
   }
 
+  /** Return the connection count of the metrics within a scope */
+  private long getConnectionCount() {
+    return connectionCount.getCount();
```

Comment:
new test in `TestMetricsConnection` to validate these counts across connection lifetimes?

Reply:
will do. thanks